contracts/vaults/Claim.sol

```diff
@@ -112,12 +122,14 @@ contract Claim is Base {
     * @param _pid The pool id
     */
     function dismissClaim(uint256 _pid) external {
+        // TODO: it's the arbitrator that dismisses the claim here, not governance
```

Comment:
should be the arbitrator only if it's challenged, no?
and challenging can be done after dismissal/approval, right?
so we need to keep the claim and "revive" it?

Reply:
no challenging can not be done after dismissal or approval (i mean, obviously not after approval, bc then we already made some payments)